cardea-server/src/main/java/ca/on/oicr/gsi/cardea/server/service/CaseService.java

```diff
@@ -165,13 +169,13 @@ private Set<String> getLimsIusIdsForShesmu(Case kase) {
         .collect(Collectors.toSet());
   }
 
-  private ShesmuCase convertCaseToShesmuCase(Case kase) {
+  private ShesmuCase convertCaseToShesmuCase(Case kase, Map<Long, Assay> assaysById) {
```

Comment:
Unnecessary parameter - this method can access `caseData.getAssaysById()` itself